cookbooks/bcpc-hadoop/recipes/resource_manager.rb

```diff
@@ -54,6 +54,9 @@
   returns [0, 1]
 end
 
+hdfs_write = "echo 'test' | hdfs dfs -copyFromLocal - /user/hdfs/chef-mapred-test"
+hdfs_remove = "hdfs dfs -rm -skipTrash /user/hdfs/chef-mapred-test"
```

Comment:
These two line may cause a compile time error.  You can execute in a resource and save the results into a ‘run_state‘ attribute to carry state